api_config.yaml

```diff
@@ -0,0 +1,21 @@
+type: google.api.Service
+config_version: 3
+
+#
+# Name of the service configuration.
+#
+name: supplyfinder.endpoints.cal-intern-project.cloud.goog
+
+#
+# API title to appear in the user interface (Google Cloud Console).
+#
+title: Supply Finder API
+apis:
+- name: supplyfinder.Finder
+- name: supplyfinder.Vendor
```

Comment:
I am not too familiar with how GCP Endpoints works, but I am pretty sure you only want to expose the Finder.  The Vendor and Supplier are implementation details.